### PR TITLE
I1 reader

### LINKED ISSUE #1: Create an draft reader class that can read an image in Hamamatsu format
See description of Hamamatsu format on OpenSlide here: https://openslide.org/formats/hamamatsu/

Some test data here: http://openslide.cs.cmu.edu/download/openslide-testdata/Hamamatsu/

### PR DESCRIPTION
**Pull request by martinjrobins**
Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The development Qt (https://www.qt.io/) libraries are required to build the GUI
 ### Ubuntu
 
 ```bash
-$ sudo apt install qt5-default
+$ sudo apt install qt5-default libopenslide-dev
```
Comment (iamleeg) [inline README.md:12]: The build failure is hard to find, I had to go from github to azure then download a compressed log file, but it eventually says:

> 2019-06-06T15:41:49.1869172Z -- Could NOT find Openslide (missing: OPENSLIDE_LIBRARIES OPENSLIDE_INCLUDE_DIRS) 

So libopenslide-dev needs to be added to the CI too.

File: cmake/FindFilesystem.cmake [hunk 2]
```diff
@@ -156,18 +156,23 @@ else()
 endif()
 
 if(_CXX_FILESYSTEM_HAVE_HEADER)
-    set(_have_fs TRUE)
+    set(_have_fs FALSE)
+    set(_is_experimenal TRUE)
```
Comment (iamleeg) [inline cmake/FindFilesystem.cmake]: typo :)

File: tests/TestSlideData.cpp [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+TEST_CASE("file invalid format", "[SlideData]")
+{
+  sight::SlideData data;
+  REQUIRE_THROWS_WITH(
+      data.open("../CMakeLists.txt"), Contains("Could not detect filetype vendor"));
+}
+
+TEST_CASE("open test file", "[SlideData]")
+{
+  // TODO: don't want to include this 200MB file in the repo....
```
Comment (iamleeg) [inline tests/TestSlideData.cpp:21]: We probably do want to include it, if the test is useful. 200MB is not so big.

**Review by iamleeg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (martinjrobins) [reply README.md:12]: Thanks, I'll add that

File: cmake/FindFilesystem.cmake (hunk 2 above)
Comment (martinjrobins) [reply cmake/FindFilesystem.cmake]: whoops.... good spot

File: tests/TestSlideData.cpp (hunk 3 above)
Comment (martinjrobins) [reply tests/TestSlideData.cpp:21]: Joshua is going to get back to us with a smaller file. The 200MB one is just one that I downloaded from the openslide web-page.

**Pull request merged**